### ISSUE TITLE
Usersnap Feedback - No way to remove a video or audio aside from manually clearing all the fields?

### ISSUE THREAD

**Issue by ascott** [participedia/usersnaps#428]
**Sender**: scottowenfletcher@gmail.com
**Comment**: No way to remove a video or audio aside from manually clearing all the fields?
[Open #133 in Usersnap Dashboard](https://usersnap.com/a/#/participedia-dev/p/participedia-201-a37e4cf6/133)

<a href='https://usersnappublic.s3.amazonaws.com/2019-05-15/01-08/667a45be-80d9-4bd6-9129-dee6402b2b01.png'>![Usersnap Feedback - No way to remove a video or audio aside from manually clearing all the fields?](https://usersnappublic.s3.amazonaws.com/2019-05-15/01-08/75957426-ff91-4b5e-af0f-394ed3a1d72a.jpg)</a>
<a href='https://usersnappublic.s3.amazonaws.com/2019-05-15/01-08/667a45be-80d9-4bd6-9129-dee6402b2b01.png'>Download original image</a>

**Browser**: Firefox 66 (Windows 10)
**Referer**: [https://ppedia-stage.herokuapp.com/case/new?full=1](https://ppedia-stage.herokuapp.com/case/new?full=1)
**Screen size**: 1600 x 900 **Browser size**: 1600 x 685

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

**Comment by jesicarson** [comment]
@ascott good point here... we should add an X beside the field (like when multiples are added) but if only one is added, clicking the X clears the field.
